.gitignore

```diff
@@ -158,3 +158,5 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 .idea/
+
+*.drawio.bkp
```

Comment:
What is this for?

Reply:
When using desktop version of Drawio, it creates backup files that should be ignored.